src/main/java/seedu/taskman/logic/commands/DeleteCommand.java

```diff
@@ -41,7 +41,7 @@ public CommandResult execute() {
         try {
             model.deleteActivity(activityToDelete);
         } catch (ActivityNotFoundException pnfe) {
-            assert false : "The target task cannot be missing";
+            assert false: "The target task cannot be missing";
```

Comment:
I'm a proponent of 'fix bad stuff when you see it' but I guess that could work too.